Upstream: yes, release branch
Reason: fix build with cmake[qt5]

From 5f210672daff7841bd83023e0d72dfb6b2dacc01 Mon Sep 17 00:00:00 2001
From: Stephen Kelly <steveire@gmail.com>
Date: Wed, 19 Mar 2014 15:34:18 +0100
Subject: [PATCH] QtDialog: Fix Qt 5 build on non-Windows.

The Qt5Core_QTMAIN_LIBRARIES variable is defined to the Qt5::WinMain
target on Windows, and undefined elsewhere.
---
 Source/QtDialog/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/QtDialog/CMakeLists.txt b/Source/QtDialog/CMakeLists.txt
index 07a50b9..7885a0c 100644
--- a/Source/QtDialog/CMakeLists.txt
+++ b/Source/QtDialog/CMakeLists.txt
@@ -28,7 +28,7 @@ if (Qt5Widgets_FOUND)
     qt5_add_resources(${ARGN})
   endmacro()
   set(CMake_QT_LIBRARIES ${Qt5Widgets_LIBRARIES})
-  set(QT_QTMAIN_LIBRARY Qt5::WinMain)
+  set(QT_QTMAIN_LIBRARY ${Qt5Core_QTMAIN_LIBRARIES})
 
   # Remove this when the minimum version of Qt is 4.6.
   add_definitions(-DQT_DISABLE_DEPRECATED_BEFORE=0)
-- 
1.9.1

